### ISSUE TITLE
Bluetooth not working after new kernel update 7.0.9

### ISSUE THREAD

**Issue by Dhanush-Projectile** [pop-os/linux#436]
The toggle in settings for bluetooth not working and it doesn't turn on even through commands

running cosmic-settings through Terminal shows 

```
> ` ERROR  Failed to read config 'accessibility_zoom', err: GetKey("accessibility_zoom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/accessibility/magnifier.rs:55 on main
> 
> 
>   ERROR  shortcuts custom config error: GetKey("custom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at /build/cosmic-settings-GVsOQ0/cosmic-settings-1.0.12~1780410623~24.04~de5e16d/vendor/cosmic-settings-config/src/shortcuts/mod.rs:43 on main
> 
>   ERROR  Failed to load Panel system config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/panel/mod.rs:91 on main
> 
>   ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/panel/inner.rs:68 on main
> 
>   ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/panel/inner.rs:68 on main
> 
>   ERROR  Failed to load Dock system config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/dock/mod.rs:135 on main
> 
>   ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/panel/inner.rs:68 on main
> 
>   ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/panel/inner.rs:68 on main
> 
>   ERROR  Failed to read config 'focus_follows_cursor', err: GetKey("focus_follows_cursor", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/window_management.rs:49 on main
> 
>   ERROR  Failed to read config 'cursor_follows_focus', err: GetKey("cursor_follows_focus", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/window_management.rs:57 on main
> 
>   ERROR  Failed to read config 'focus_follows_cursor_delay', err: GetKey("focus_follows_cursor_delay", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/window_management.rs:66 on main
> 
>   ERROR  Failed to read config 'active_hint', err: GetKey("active_hint", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/window_management.rs:75 on main
> 
>   ERROR  Failed to read config 'edge_snap_threshold', err: GetKey("edge_snap_threshold", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/window_management.rs:84 on main
> 
>   ERROR  shortcuts custom config error: GetKey("custom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at /build/cosmic-settings-GVsOQ0/cosmic-settings-1.0.12~1780410623~24.04~de5e16d/vendor/cosmic-settings-config/src/shortcuts/mod.rs:43 on main
> 
>   ERROR  Failed to read config 'workspaces', err: GetKey("workspaces", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/desktop/workspaces.rs:40 on main
> 
>   ERROR  Failed to read config 'input_default', why: GetKey("input_default", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/input/mod.rs:51 on main
> 
>   ERROR  Failed to read config 'workspaces', err: GetKey("workspaces", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/input/touchpad.rs:26 on main
> 
>   ERROR  Failed to read config 'descale_xwayland', err: GetKey("descale_xwayland", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/applications/legacy_applications.rs:53 on main
> 
>   ERROR  Failed to read config 'xwayland_eavesdropping', err: GetKey("xwayland_eavesdropping", Os { code: 2, kind: NotFound, message: "No such file or directory" })
>     at cosmic-settings/src/pages/applications/legacy_applications.rs:62 on main
> 
> active
> enabled
>   ERROR  Bluetooth object watcher has shutdown unexpectedly
>     at subscriptions/bluetooth/src/subscription.rs:136 on tokio-rt-worker
> 
> active
> enabled
>   ERROR  Bluetooth object watcher has shutdown unexpectedly
>     at subscriptions/bluetooth/src/subscription.rs:136 on tokio-rt-worker`
> ```
```

after runnnig journalctl -eu bluetooth.service show

`Jun 04 14:38:53 pop-os systemd[1]: Starting bluetooth.service - Bluetooth service...
Jun 04 14:38:53 pop-os (uetoothd)[902]: bluetooth.service: ConfigurationDirectory 'bluetooth' already e>
Jun 04 14:38:53 pop-os bluetoothd[902]: Bluetooth daemon 5.72
Jun 04 14:38:53 pop-os bluetoothd[902]: Starting SDP server
Jun 04 14:38:53 pop-os systemd[1]: Started bluetooth.service - Bluetooth service.
Jun 04 14:38:53 pop-os bluetoothd[902]: src/plugin.c:plugin_init() System does not support csip plugin
Jun 04 14:38:53 pop-os bluetoothd[902]: profiles/audio/micp.c:micp_init() D-Bus experimental not enabled
Jun 04 14:38:53 pop-os bluetoothd[902]: src/plugin.c:plugin_init() System does not support micp plugin
Jun 04 14:38:53 pop-os bluetoothd[902]: src/plugin.c:plugin_init() System does not support vcp plugin
Jun 04 14:38:53 pop-os bluetoothd[902]: src/plugin.c:plugin_init() System does not support mcp plugin
Jun 04 14:38:53 pop-os bluetoothd[902]: src/plugin.c:plugin_init() System does not support bass plugin
Jun 04 14:38:53 pop-os bluetoothd[902]: src/plugin.c:plugin_init() System does not support bap plugin
Jun 04 14:38:53 pop-os bluetoothd[902]: Bluetooth management interface 1.23 initialized`

`[    0.111684] Spectre V2 : Enabling Speculation Barrier for firmware calls
[    0.312714] ACPI: [Firmware Bug]: BIOS _OSI(Linux) query ignored
[    0.348929] acpi PNP0A08:00: [Firmware Info]: ECAM [mem 0xf8000000-0xfbffffff] for domain 0000 [bus 00-3f] only partially covers this bridge
[    5.011398] amdgpu 0000:03:00.0: [drm] Loading DMUB firmware via PSP: version=0x0101002B
[    5.012054] amdgpu 0000:03:00.0: [VCN instance 0] Found VCN firmware Version ENC: 1.24 DEC: 8 VEP: 0 Revision: 9
[    7.847849] Bluetooth: Core ver 2.22
[    7.847884] NET: Registered PF_BLUETOOTH protocol family
[    7.847887] Bluetooth: HCI device and connection manager initialized
[    7.847892] Bluetooth: HCI socket layer initialized
[    7.847896] Bluetooth: L2CAP socket layer initialized
[    7.847901] Bluetooth: SCO socket layer initialized
[    8.002916] usbcore: registered new interface driver btusb
[    8.008694] Bluetooth: hci0: HW/SW Version: 0x008a008a, Build Time: 20251223091725
[    8.042350] mt7921e 0000:01:00.0: enabling device (0000 -> 0002)
[    8.048418] mt7921e 0000:01:00.0: ASIC revision: 79610010
[    8.131227] Bluetooth: hci0: Failed to send wmt func ctrl (-22)
[    8.131235] Bluetooth: hci0: HCI Enhanced Setup Synchronous Connection command is advertised, but not supported.
[    8.227992] mt7921e 0000:01:00.0: HW/SW Version: 0x8a108a10, Build Time: 20251223091050a
[    8.239694] mt7921e 0000:01:00.0: WM Firmware Version: ____010000, Build Time: 20251223091148
[    8.587945] Bluetooth: BNEP (Ethernet Emulation) ver 1.3
[    8.587958] Bluetooth: BNEP filters: protocol multicast
[    8.587974] Bluetooth: BNEP socket layer initialized
[    9.075356] mt7921e 0000:01:00.0 wlp1s0: renamed from wlan0
[ 3328.339455] usbcore: deregistering interface driver btusb
[ 3328.571070] usbcore: registered new interface driver btusb
[ 3328.574152] Bluetooth: hci0: HW/SW Version: 0x008a008a, Build Time: 20251223091725
[ 3328.697610] Bluetooth: hci0: Failed to send wmt func ctrl (-22)
[ 3328.697632] Bluetooth: hci0: HCI Enhanced Setup Synchronous Connection command is advertised, but not supported.
[ 3337.980903] Bluetooth: hci0: HW/SW Version: 0x008a008a, Build Time: 20251223091725
[ 3338.105333] Bluetooth: hci0: Failed to send wmt func ctrl (-22)
[ 3338.105357] Bluetooth: hci0: HCI Enhanced Setup Synchronous Connection command is advertised, but not supported.`

after searching alot i find out kernel 7.0.9 bricked mediatek adapters for bluetooth

**Comment by vdubgeek** [comment]
Same issue is occuring on my laptop with the new kernel, after updating to COSMIC Epoch 1.0.15

**Comment by mmstick** [comment]
Not related to COSMIC. If you have a Mediatek card, it may be fixed by Linux 7.0.11 which is currently being built for testing.

**Comment by Dhanush-Projectile** [comment]
Is there any way to install kernel 7.0.10

**Comment by leviport** [comment]
tracking in https://github.com/pop-os/linux/issues/431

7.0.10 is coming soon
